blog/arquillian-cube-1.5.0.textile

```diff
@@ -0,0 +1,20 @@
+---
+date: 2017-05-30
+author: lordofthejars
+---
+
+p. New version of Arquillian Cube 1.5.0 has been released with new features and bug fixes.
+
+h3. What's new
+
+p. <i class="icon-star"></i> **Set System Properties with Docker container information**
+
+p. When running a test next System Properties should be available:
+
+* @arq.cube.docker.host@ setting the docker host ip.
+
+p. And for each docker container defined:
+
+* @arq.cube.docker.<cubeid>.ip@ with container ip.
```

Comment:
Maybe underscore things like `<cubeid>` to `<cube_id>` - reads a bit better IMHO